app/src/util/validation.ts

```diff
@@ -116,6 +116,7 @@ export type CreatePopulationRequest = z.infer<typeof createPopulationRequest>;
 // user file schema validation
 export const createUserFileRequset = z.object({
   userId: z.string().uuid().optional(),
+  cityId: z.string().uuid().optional(),
```

Comment:
Tbh I think userId and cityId are mandatory, because without them we can't really reference the files anymore. So would be fine to remove `.optional()` here.